src/components/Field/Field.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Field = ({
+  name, err, handleChange, validation, value, valid,
+}) => {
+  let option;
+  let classTitle;
+
+  if (name.includes('Url')) {
+    option = 'url';
+  } else if (name === 'title' || name === 'imdbId') {
+    option = 5;
+  } else {
+    option = 30;
+  }
+
+  if (valid === false) {
+    classTitle = 'no-verified';
+  } else if (valid === true) {
+    classTitle = 'accepted';
+  } else {
+    classTitle = 'before-verification';
+  }
+
+  return (
+    <div>
+      <input
+        className={classTitle}
+        value={value}
+        placeholder={name}
+        type="text"
+        name={`${name}`}
+        onChange={ev => handleChange(ev.target.name, ev.target.value)}
```

Comment:
не сокращай названия евента
рекомендую прокидывать функцию, а в ней уже доставать из ивента что тебе нужно
```suggestion
        onChange={handleChange}
```